src/apps/service/apps/job_listings.clj

```diff
@@ -70,7 +70,8 @@
   [{user-id :user_id :as job} rep-steps]
   (let [interactive? (fn [step] (= (:job_type step) jp/interactive-job-type))
         get-url      (fn [step] (str (interapps-url (url (config/interapps-base)) user-id (:external_id step))))]
-    (seq (map get-url (filter interactive? (rep-steps (:id job)))))))
+    (when-not (:is_batch job)
+      (seq (map get-url (filter interactive? (rep-steps (:id job))))))))
```

Comment:
With the previous query, no interactive steps could ever have been associated with a batch parent job (even if interactive batches were allowed). With the new query, interactive steps would appear in a batch containing interactive jobs, so checking to see if the job is a batch first prevents batch parent jobs from having interactive URLs associated with them. Admittedly, interactive jobs still aren't allowed in batches, but I figured that it wouldn't hurt to future-proof this code a bit.